final-project/src/components/TooltipPopup/TooltipPopup.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PopupWithForm from "../PopupWithForm/PopupWithForm";
+
+function TooltipPopup({ isOpen, onClose, handleToolTipToggle }) {
+
+  return (
+    <PopupWithForm isOpen={isOpen} onClose={onClose}>
```

Comment:
`PopupWithForm` is not suitable for `TooltipPopup` because it doesn't have any `form`. So, the name is misleading